### PR TITLE
[2.7.2] Release System Charts

### PR DESCRIPTION
**Pull request by eliyamlevy**

### CODE REVIEW

File: charts/rancher-k3s-upgrader/0.4.0/questions.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.6.0-alpha1
```
Comment (eliyamlevy) [inline charts/rancher-k3s-upgrader/0.4.0/questions.yml:1]: @Oats87 @HarrisonWAffel, the `questions.yml` doesn't have a max constraint.

Comment (prachidamle) [reply charts/rancher-k3s-upgrader/0.4.0/questions.yml:1]: confirmed by @jiaqiluo  this is not a release-blocker and should be ok to be fixed in the next release.

**Review by pennyscissors — APPROVED** [review]
LGTM, needs @prachidamle  to approve and merge

**Review by prachidamle — COMMENTED** [review]
LGTM

**Pull request merged**